libkbfs/reporter_kbpki.go

```diff
@@ -230,23 +232,37 @@ func (r *ReporterKBPKI) NotifySyncStatus(ctx context.Context,
 	}
 }
 
+// SuppressNotifications implements the Reporter interface for ReporterKBPKI.
+func (r *ReporterKBPKI) SuppressNotifications(
+	ctx context.Context, suppressDuration time.Duration) {
+	r.suppressCh <- suppressDuration
+}
+
 // Shutdown implements the Reporter interface for ReporterKBPKI.
 func (r *ReporterKBPKI) Shutdown() {
 	r.canceler()
 	close(r.notifyBuffer)
 	close(r.notifySyncBuffer)
+	close(r.suppressCh)
```

Comment:
This is a problem in all the other channels as well, but after `Shutdown`, any calls to `SuppressNotifications` will panic. It's probably fine, just something to consider.

Reply:
It's already shut down, so yeah not really a big issue. I guess we can just leave them all open if care?

Reply:
No they need to be closed so the loops finish (or we'd need a shutdown channel otherwise). I'm just wary of races where something calls into SimpleFS's `SuppressNotifications` after shutdown has been called.

Reply:
The canceler should release the loop, so ctx.Done() is sort of a shutdown channel here. Although it seems we are not checking it now (despite of passing it in). I'll put up a PR to fix it.